helix-term/src/commands.rs

```diff
@@ -3384,47 +3385,41 @@ fn select_register(cx: &mut Context) {
     })
 }
 
-fn view_mode(cx: &mut Context) {
-    cx.on_next_key(move |cx, event| {
-        if let KeyEvent {
-            code: KeyCode::Char(ch),
-            ..
-        } = event
-        {
-            // if lock, call cx again
-            // TODO: temporarily show VIE in the mode list
-            match ch {
-                // center
-                'z' | 'c'
-                // top
-                | 't'
-                // bottom
-                | 'b' => {
-                    let (view, doc) = current!(cx.editor);
+fn align_view_top(cx: &mut Context) {
+    let (view, doc) = current!(cx.editor);
+    align_view(doc, view, Align::Top);
+}
 
-                    align_view(doc, view, match ch {
-                        'z' | 'c' => Align::Center,
-                        't' => Align::Top,
-                        'b' => Align::Bottom,
-                        _ => unreachable!()
-                    });
-                }
-                'm' => {
-                    let (view, doc) = current!(cx.editor);
-                    let pos = doc.selection(view.id).cursor();
-                    let pos = coords_at_pos(doc.text().slice(..), pos);
+fn align_view_center(cx: &mut Context) {
+    let (view, doc) = current!(cx.editor);
+    align_view(doc, view, Align::Center);
+}
 
-                    const OFFSET: usize = 7; // gutters
-                    view.first_col = pos.col.saturating_sub(((view.area.width as usize).saturating_sub(OFFSET)) / 2);
-                },
-                'h' => (),
-                'j' => scroll(cx, 1, Direction::Forward),
-                'k' => scroll(cx, 1, Direction::Backward),
-                'l' => (),
-                _ => (),
-            }
-        }
-    })
+fn align_view_bottom(cx: &mut Context) {
+    let (view, doc) = current!(cx.editor);
+    align_view(doc, view, Align::Bottom);
+}
+
+// TODO: fix naming
+fn view_what(cx: &mut Context) {
```

Comment:
This is middle as in the middle of the line (horizontal centering)

Reply:
But it didn't go to the middle for me.